core/schema_utils.py

```diff
@@ -681,3 +681,21 @@ def is_valid_username_string(obj: str) -> bool:
             return True
         except utils.ValidationError:
             return False
+
+    @staticmethod
+    def is_allowed_audio_extension(obj: str) -> bool:
```

Comment:
```suggestion
    def has_allowed_audio_extension(obj: str) -> bool:
```

Reply:
Also, please add tests for this function.

Reply:
Done.